bcda/auth/provider.go

```diff
@@ -2,14 +2,59 @@ package auth
 
 import (
 	"os"
+	"strings"
 
-	jwt "github.com/dgrijalva/jwt-go"
+	"github.com/dgrijalva/jwt-go"
+	log "github.com/sirupsen/logrus"
 )
 
-// Provider is an interface for operations performed through an authentication provider.
+const (
+	Alpha = "alpha"
+	Okta  = "okta"
+)
+
+var providerName = Alpha
+
+func init () {
+	log.SetFormatter(&log.JSONFormatter{})
+	SetProvider(strings.ToLower(os.Getenv(`BCDA_AUTH_PROVIDER`)))
+}
+
+func SetProvider(name string) {
```

Comment:
should this return the provider as well for convenience?

Reply:
I think having the set of a get/set pair return a value would be unexpected. I was trying to keep the mechanics similar to setting an env var to select the auth provider, since that's the standard use case. In other words, you set the type name, and then that's what you get from that point forward.